siphon/ncss.py

```diff
@@ -2,6 +2,8 @@
 from io import BytesIO
 
 import numpy as np
+from os import remove
+import atexit
```

Comment:
We group the imports (based partially on PEP8) by standard library (first full import, then partial import), then third party, then our own; arranged alphabetically. Doing it consistently makes it easy to find stuff when you have quite a few imports.